### ISSUE TITLE
Use with pipe?

### ISSUE THREAD

**Issue by MikeMichel** [Nathan219/docker-stream-cleanser#4]
Hi,

im trying to get this work in a pipe request where multiple containerlogs end in one. Right now i get the unwanted hex response. 

```
              req.pipe(request.get({
                url: 'http://' + appArray[i].dockerHost + ':4243' + config.DOCKERLOGPART1 + appArray[i].dockerId + '/logs?stderr=1&stdout=1&tail=10',
                qs: req.query
              }, function (error, response, body) {
                if (error) {
                  console.error('Connection error: ' + error.code);
                } if (response){
                  console.log('Response: ' + response);
                  //Object
                }if (body) {
                console.log('Body: ' + body);
                // looks good
                }
                
              })).pipe(res)
//binary hex stuff
```
Can you give me a hint how to use your function to get the clean ouput?

Best,

Mike

**Comment by tjmehta** [comment]
new api coming with #5

**Comment by tjmehta** [comment]
Released v0.2.0
